js/form_validation.js

```diff
@@ -15,6 +14,6 @@ formButton.addEventListener('click', (e) => {
     errorMsg.classList.add('show_email_error_msg');
   } else {
     errorMsg.classList.remove('show_email_error_msg');
-    form.submit();
+    window.location.href = 'https://formspree.io/f/xbjwdjrr';
```

Comment:
- [ ] Based on the project requirements of this milestone, the contact form contains at least one validation checking if the email is written in all lowercase. If a user submits the contact form with the email field containing uppercase characters an error message should show up. Kindly, implement it!

Reply:
hi @elyor-doniyorov 
thanks for reviewing i have the lower case validation in my project, so if you can clarify so that i undestand what to change i will appreciate
![form_validation_red](https://user-images.githubusercontent.com/59707859/151391861-1228ba23-d507-4b4d-8afd-e29a527542be.PNG)